src/YearsView.js

```diff
@@ -31,16 +31,25 @@ var DateTimePickerYears = React.createClass({
         var currentYear, disabled,
             // Month and date are irrelevant here because
             // we're only really interested in the year
-            irrelevantMonth = 1,
+            irrelevantMonth = 0,
```

Comment:
That's a good question. It seems that momentjs considers that months can be set from 0 to 11 when we use the set method. Here is the corresponding doc: http://momentjs.com/docs/#/get-set/set/.